department-of-veterans-affairs/abd-vro
# MCP-2129 Checkstyle error fixes

## What was the problem?
Code formatting checkstyle errors are present in the build logs when they should not be.

Associated tickets or Slack threads:

- [MCP-2129](https://amida.atlassian.net/browse/MCP-2129)

## How does this fix it?[^1]
Gets rid of checkstyle code formatting errors in the build logs

## How to test this PR
- Step1: build project
- Step 2: verify checkstyle errors are gone


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
